.github/workflows/common.yml

```
@@ -0,0 +1,32 @@
+        required: false
+        type: string
+        default: "17"
+
+env:
+  # Common environment variables
+  NODE_VERSION: ${{ inputs.node-version || '18' }}
+  JAVA_VERSION: ${{ inputs.java-version || '17' }}
+  GRADLE_OPTS: -Dorg.gradle.daemon=false
+  GRADLE_USER_HOME: ${{ github.workspace }}/.gradle
+
+jobs:
```

actionlint (1.7.8)

"jobs" section should not be empty. please remove this section if it's unnecessary

(syntax-check)